pkg/server/plugin/notifier/k8sbundle/k8sbundle_test.go

```diff
@@ -167,44 +128,37 @@ func (s *Suite) TestBundleLoadedConfigMapUpdateConflict() {
 	})
 
 	// return a different bundle when fetched the second time
-	s.r.AppendBundle(testBundle)
-	s.r.AppendBundle(testBundle2)
+	test.identityProvider.AppendBundle(testBundle)
+	test.identityProvider.AppendBundle(testBundle2)
 
-	s.configure("")
-
-	resp, err := s.p.NotifyAndAdvise(context.Background(), &notifierv0.NotifyAndAdviseRequest{
-		Event: &notifierv0.NotifyAndAdviseRequest_BundleLoaded{
-			BundleLoaded: &notifierv0.BundleLoaded{
-				Bundle: testBundle,
-			},
-		},
-	})
-	s.NoError(err)
-	s.AssertProtoEqual(&notifierv0.NotifyAndAdviseResponse{}, resp)
+	bundle := &common.Bundle{
```

Comment:
We seem to be repeating this bundle quite a bit... should we lift this up to a package level var?